Module/MyPythonNode.cs

```diff
@@ -148,9 +191,17 @@ public class MyPythonNode : MyScriptableNode, IMyVariableBranchViewNodeBase
 
         public class Node
         {
+            public struct DashboardProxy
+            {
+                public PythonDictionary set;
+            };
+
             //global comunucation channel between python-scipt nodes
             public static PythonDictionary blackboard;
 
+            //dashboard change data
+            public DashboardProxy dashboard;
```

Comment:
Why not just public PythonDictionary Dashboard? The struct is always only used for accessing the "set" field. Also, public members should have capital first letter.